### PR TITLE
python3Packages.latexify-py: init at 0.2.0

### PR DESCRIPTION
**Pull request by prusnak**
###### Description of changes

- new package

FhkFrL_UYAILNkx

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by dotlambda — CHANGES_REQUESTED** [review]
Please add `pytestCheckHook` to `checkInputs`.

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -5055,6 +5055,8 @@ self: super: with self; {
 
   latexcodec = callPackage ../development/python-modules/latexcodec { };
 
+  latexify = callPackage ../development/python-modules/latexify { };
```
Comment (dotlambda) [inline pkgs/top-level/python-packages.nix:5058]:
```suggestion
  latexify-py = callPackage ../development/python-modules/latexify-py { };
```

File: pkgs/development/python-modules/latexify/default.nix [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, hatchling
+, dill
+}:
+
+buildPythonPackage rec {
+  pname = "latexify";
```
Comment (dotlambda) [inline pkgs/development/python-modules/latexify/default.nix:9]:
```suggestion
  pname = "latexify-py";
```

**Review by dotlambda — CHANGES_REQUESTED** [review]
This is how it's called on PyPI.

**Comment by dotlambda** [comment]
> > Please add `pytestCheckHook` to `checkInputs`.
> 
> When I do so, I hit the following issue which I am unable to fix :-/

Try `cd src` in `preCheck`.

File: pkgs/development/python-modules/latexify/default.nix (hunk 2 above)
Comment (prusnak) [reply pkgs/development/python-modules/latexify/default.nix]: Done in bbdd1913fa6474e78da9e0cc2184287c3a8bf0f0

**Comment by prusnak** [comment]
> Try `cd src` in `preCheck`.

Thanks, that did the trick: added in 1cd546a22b6506156087e4b4d5bc2508223f72e2

**Pull request merged**